Assets/Scripts/UI/UIManager.cs

```diff
@@ -13,19 +15,34 @@ public class UIManager : ManageableObject
         public Image lockOnImage;
 
         public UITracker grappleReticule;
-
         public Image grappleImage;
 
         public AgentHealthBarsPool agentHealthBarsPool;
 
         public UIMeter playerHealth;
 
+        protected List<UITracker> uiTrackers;
```

Comment:
We probably don't need to differentiate between the lockon and grapple ui trackers and the others. Everything can just go in uiTrackers

Reply:
I think that'd be cleaner. There was not a way to get it to merge cleanly without doing it this way though. I'm all for fixing it now.